app/section/sections/browser-additional-information/ui.test.js

```diff
@@ -0,0 +1,181 @@
+
+  const submitButton = Selector('[data-test-id="section-submit-button"]');
+
+  await t
+    .expect(submitButton.find('button').count).eql(1);
+});
+
+test('should show error summary and validation for questions when they exceed the maxLength', async (t) => {
+  await pageSetup(t);
+
+  nock('http://localhost:8080')
+    .put('/api/v1/Solutions/S100000-001/sections/browser-additional-information')
+    .reply(400, {
+      maxLength: ['additional-information'],
+    });
+
+  const oneHundredCharacters = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789';
+  const thousandCharacters = oneHundredCharacters.repeat(10);
+
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const errorSummaryList = Selector('.nhsuk-error-summary__list');
+  const additionalInformation = Selector('[data-test-id="question-additional-information"]');
+  
+  const submitButton = Selector('[data-test-id="section-submit-button"] button');
+  
+  await t
+  .expect(errorSummary.exists).notOk()
+  .typeText(additionalInformation, `${thousandCharacters}0`, { paste: true })
+  .click(submitButton)
+  .expect(errorSummary.exists).ok()
+  .expect(errorSummaryList.find('li').count).eql(1)
+  .expect(errorSummaryList.find('li:nth-child(1)').innerText).eql('Additional information is over the character limit')
+  .expect(errorSummaryList.find('li:nth-child(1) a').getAttribute('href')).eql('#additional-information')
+  .expect(additionalInformation.find('.nhsuk-textarea--error').exists).ok()
+  .expect(additionalInformation.find('.nhsuk-error-message').innerText).eql('Error:\nAdditional information is over the character limit')
+});
+
+test('should goto anchor when clicking the additional information required error link', async (t) => {
+  await pageSetup(t);
+
+  nock('http://localhost:8080')
+    .put('/api/v1/Solutions/S100000-001/sections/browser-additional-information')
+    .reply(400, {
+      maxLength: ['additional-information'],
+    });
+
+  const oneHundredCharacters = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789';
+  const thousandCharacters = oneHundredCharacters.repeat(10);
+
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const errorSummaryList = Selector('.nhsuk-error-summary__list');
+  const additionalInformation = Selector('[data-test-id="question-additional-information"]');
+  
+  const submitButton = Selector('[data-test-id="section-submit-button"] button');
+
+  const getLocation = ClientFunction(() => document.location.href);
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .typeText(additionalInformation, `${thousandCharacters}0`, { paste: true })
+    .click(submitButton)
+    .expect(errorSummary.exists).ok()
+    .expect(errorSummaryList.find('li:nth-child(1) a').count).eql(1)
+    .expect(errorSummaryList.find('li:nth-child(1) a').getAttribute('href')).eql('#additional-information')

```

Comment:
I would just test the anchor behavior. The above is already covered in the test above. Just keep the `click(submitButton)`